framework/src/org/checkerframework/common/value/ValueCheckerUtils.java

```diff
@@ -97,7 +98,8 @@
         } else if (AnnotationUtils.areSameByClass(anno, BottomVal.class)) {
             values = convertBottomVal(anno, castType);
         } else if (AnnotationUtils.areSameByClass(anno, UnknownVal.class)
-                || AnnotationUtils.areSameByClass(anno, ArrayLen.class)) {
+                || AnnotationUtils.areSameByClass(anno, ArrayLen.class)
+                || AnnotationUtils.areSameByClass(anno, IntRange.class)) {
```

Comment:
If converting from small `@IntRange` to `@IntVal` is not urgently needed, I'd agree with what you say. Thanks.